BREAKING.md

```diff
@@ -105,6 +106,9 @@ They are deprecated from `@fluidframework/core-interface` and would be removed i
 ### Deprecated `IFluidSerializer` in `IFluidDataStoreRuntime`
 `IFluidSerializer` should only be used by DDSs to serialize data and they should use the one created by `SharedObject`.
 
+### Removed `readAndParseFromBlobs` from `driver-utils`
```

Comment:
sounds like @timtwang is running the release right now so make sure  to keep an eye on that so this is in the right version section if this goes in after that change